project/accounts/api.py

```diff
@@ -0,0 +1,221 @@
+        """ """
+        account = get_account(username=user__username)
+        if self.request.user == account.user:
+            serializer = AccountSerializer(account)
+        else:
+            serializer = AccountListSerializer(account)
+        return Response(serializer.data)
+
+    @action(detail=True)
+    def civis(self, request, user__username=None):
+        """
+        Gets the civis of the selected account
+        /accounts/{username}/civis
+        """
+        account = get_account(username=user__username)
+        account_civis = account.civis
+        serializer = CiviSerializer(account_civis, many=True)
+        return Response(serializer.data)
+
+    @action(detail=True)
+    def followers(self, request, user__username=None):
+        """
+        Gets the followers of the selected account
+        /accounts/{username}/followers
+        """
+        account = get_account(username=user__username)
+        account_followers = account.followers.all()
+        serializer = AccountListSerializer(account_followers, many=True)
+        return Response(serializer.data)
+
+    @action(detail=True)
+    def following(self, request, user__username=None):
+        """
+        Gets the followings of the selected account
+        /accounts/{username}/following
+        """
+        account = get_account(username=user__username)
+        account_followings = account.following.all()
+        serializer = AccountListSerializer(account_followings, many=True)
+        return Response(serializer.data)
+
+    @action(detail=True)
+    def categories(self, request, user__username=None):
+        """
+        Gets the preferred categories of the selected account
+        /accounts/{username}/categories
+        """
+        account = get_account(username=user__username)
+        account_categories = account.categories
+        serializer = CategorySerializer(account_categories, many=True)
+        return Response(serializer.data)
+
+    @action(detail=True)
+    def threads(self, request, user__username=None):
+        """
+        Gets the preferred categories of the selected account
+        /accounts/{username}/categories
+        """
+        account = get_account(username=user__username)
+        draft_threads = Thread.objects.filter(author=account).exclude(is_draft=False)
+        serializer = ThreadSerializer(
+            draft_threads, many=True, context={"request": request}
+        )
+        return Response(serializer.data)
+
+    @action(detail=True)
+    def drafts(self, request, user__username=None):
+        """
+        Gets the draft threads of the selected account
+        /accounts/{username}/drafts
+        """
+        account = get_account(username=user__username)
+        draft_threads = Thread.objects.filter(author=account, is_draft=False)
+        serializer = ThreadSerializer(
+            draft_threads, many=True, context={"request": request}
+        )
+        return Response(serializer.data)
```

Comment:
Add a blank line at end of file
```suggestion
        return Response(serializer.data)

```